README.md

```diff
@@ -1,6 +1,20 @@
-# Default Optimizely Worker Template.
+# Optimizely Full Stack Feature Flags and Experimentation
 
-This is the default optimizely starter template for cloudflare workers. This template provides an npm package with starter code and required dependencies.
+[Optimizely Full Stack](https://docs.developers.optimizely.com/full-stack/docs) is a feature flagging and experimentation platform for websites, mobile apps, chatbots, APIs, smart devices, and anything else with a network connection.
```

Comment:
Might want to double check with Trisha regarding the documentation links.

Reply:
@trishahanlon It will be great if you can take a look and confirm if its the correct link

Reply:
@opti-jnguyen and @zashraf1985  Good call! I added a suggested change with the new doc url. Thanks!